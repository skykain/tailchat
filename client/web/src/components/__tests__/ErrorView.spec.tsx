import { render } from '@testing-library/react';
import React from 'react';
import { ErrorView } from '../ErrorView';

describe('ErrorView', () => {
  test('render', () => {
    const wrapper = render(<ErrorView error={new Error('Dummy Error')} />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
