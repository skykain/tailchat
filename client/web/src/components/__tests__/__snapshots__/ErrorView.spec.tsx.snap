// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorView render 1`] = `
<div>
  <div
    class="text-center w-full"
  >
    <img
      class="w-32 h-32 m-auto mb-2"
      src="problem.svg"
    />
    <div>
      Dummy Error
    </div>
  </div>
</div>
`;
